scripts/ci/check-destructive-migrations.js

```
@@ -0,0 +1,263 @@
+function getChangedMigrationFiles(changedFiles) {
+  return changedFiles
+    .map(normalizePath)
+    .filter((filePath) => filePath.startsWith(MIGRATIONS_ROOT) && filePath.endsWith('.sql'));
+}
+
+function splitSqlStatements(sql) {
+  return sql
+    .split(/;\s*(?:\r?\n|$)/)
+    .map((statement) => statement.trim())
+    .filter(Boolean)
+    .map((statement) => (statement.endsWith(';') ? statement : `${statement};`));
```

ESLint

[error] Replace `·(statement.endsWith(';')·?·statement·:·`${statement};`)` with `⏎······statement.endsWith(";")·?·statement·:·`${statement};`,⏎····`

(prettier/prettier)